components/CardTrombinoscope.vue

```diff
@@ -1,73 +1,77 @@
 <template>
-  <div class="card">
-    <header class="card-header">
-      <p class="card-header-title is-centered">
-        {{ poste }}
-      </p>
-    </header>
-    <div class="card-image">
-      <div>
-        <p>
-          <strong>{{ name }}</strong
-          ><br /><br /><br />
-          <a v-if="url" :href="url">Lien: Facebook</a>
+    <div class="card">
+      <header class="card-header">
+        <p class="card-header-title is-centered">
+          {{poste}}
         </p>
+      </header>
+      <div class="card-image">
+        <div>
+            <p>
+              <strong>{{name}}</strong><br>{{ description }}<br>
+           <a v-if="url" :href="url">Lien: Facebook</a> </p>
+        </div>
+        <figure class="image is-2by3">
+          <img v-if="picture" :src="require(`~/assets/img/${picture}`)" :alt="`${picture}`">
+        </figure>
       </div>
-      <figure class="image is-2by3">
-        <img
-          :src="require(`~/assets/trombinoscope/${picture}`)"
-          :alt="`Photo du trombinoscope du ${poste}`"
-        />
-      </figure>
     </div>
-  </div>
+<<<<<<< HEAD
+=======
+
+>>>>>>> 41dfcf0e46993ce32209000957717c9235a16d42
 </template>
```

Comment:
### **This needs to be fixed.**

Reply:
Resolved in https://github.com/BDE-Polytech-MTP/website-front/pull/66/commits/d1bcb0eba1d3fc0a4625cddf5f5d602a7a60fbb1